Mathlib/Topology/Maps/Basic.lean

```diff
@@ -363,6 +363,12 @@ theorem of_isEmpty [h : IsEmpty X] (f : X → Y) : IsOpenMap f := of_nhds_le h.e
 
 end IsOpenMap
 
+/-- The preimage of a `Dense` set under a continuous open map is `Dense`. -/
+theorem Dense.dense_preimage_of_isOpenMap (hf : IsOpenMap f) (hfc : Continuous f)
+    {t : Set Y} (ht : Dense t) : Dense (f ⁻¹' t) := by
```

Comment:
You don't need continuity here, see #15552

Reply:
Thanks, I merged #15552 and deleted my version